policy/release/slsa_build_build_service/slsa_build_build_service.rego

```diff
@@ -103,4 +103,10 @@ _rule_data_errors contains error if {
 	}
 }
 
+_builder_id(att) := builder_id if {
```

Comment:
Maybe add a little commentary here to explain which SLSA version each part is supposed to work for.